Student-Unity-frontend/src/app/components/announcement/announcement.component.ts

```diff
@@ -34,4 +34,8 @@ export class AnnouncementComponent implements OnInit {
         let uniqueId = randomLetter + Date.now();
         return uniqueId;
     }
+
+  removeTask() {
```

Comment:
what's removeTask do here in announcement page

Reply:
no thing I was planning to do remove announcement like remove  task  but I changed My idea.
I did it in the back only

Reply:
we may do it later